### PR TITLE
[W5.3b][W13-A1]Raynold Ng 

### PR DESCRIPTION
**Pull request by raynoldng**
LO-Inheritance

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:3]: This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    public String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:4]: Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public int hashCode() {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:21]: In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!

**Review by okkhoy — COMMENTED** [review]
You can try to move the validation logic as well to the superclass.

**Comment by okkhoy** [comment]
@raynoldng 
Some comments added. Please ack & close the PR after reading comments.

**Comment by raynoldng** [comment]
Acknowledged with thanks

**Pull request closed without merging**